yt/frontends/ramses/tests/test_outputs.py

```diff
@@ -437,4 +437,16 @@ def test_ramses_empty_record():
     ds.index
 
     # Access some field
-    ds.r[('gas', 'density')]
\ No newline at end of file
+    ds.r[('gas', 'density')]
+
+output_00080 = "output_00080/info_00080.txt"
+@requires_ds(output_00080)
```

Comment:
Because this test uses the f90nml module I think you need the `@requires_module()` decorator.

Reply:
Ha nice mind-meld with @Xarthisius! 😎